codeowners_diff.py

```diff
@@ -16,6 +16,9 @@
 from tabulate import tabulate
 
 
+CURRENT_STRING = 'CURRENT'
```

Comment:
Alternatives might be "UNCOMMITTED" or maybe just ".".

Reply:
Agreed that it *can* be confusing when thinking about it, but it feels very natural when actually using it.

Open to changing it to something else :) I initially added a separate flag as suggested but it as good a UX, and left me wondering about things like which should take precedence etc.

> Current means something the me here as in my neovim config I have an install script which when passed `-c` or `--current` it will install the current on disk version rather than checking out the latest version/tag. It's "what i am currently working on" hence the use of the word `current`.